tests/unit_tests/test_mcda_with_robustness.py

```diff
@@ -68,30 +72,29 @@ def get_list_random_input_matrices() -> list[pd.DataFrame]:
         data3 = np.random.normal(loc=9.5, scale=0.1, size=(4, 10))
         data3 = pd.DataFrame(data3)
         out_list = [data1, data2, data3]
-        print('out_list', out_list)
 
-        input_matrix = TestMCDA_with_variability.get_input_matrix()
-        config = TestMCDA_with_variability.get_test_config()
+        input_matrix = TestMCDA_with_robustness.get_input_matrix()
+        config = TestMCDA_with_robustness.get_test_config()
         config = Config(config)
-        MCDA_w_var = MCDAWithVar(config, input_matrix)
-        output_list = MCDA_w_var.convert_list(out_list)
+        MCDA_w_uncert = MCDAWithRobustness(config, input_matrix)
```

Comment:
It is slightly confusing to me if `_w_` is `with` or `without`.
If I see it at the place of initialisation, then it is clear that it is `with` but reading it down in the code leads to confusion.

Reply:
Solved. I have refactored all in `mcda_with_robustness`.